searx/autocomplete.py

```diff
@@ -252,11 +253,36 @@ def yandex(query, _lang):
 }
 
 
-def search_autocomplete(backend_name, query, sxng_locale):
-    backend = backends.get(backend_name)
-    if backend is None:
-        return []
+def search_autocomplete(backend_names, query, sxng_locale):
+
+    enabled_backends = set()
+    for name in backend_names:
+        backend = backends.get(name)
+        if backend is None:
+            logger.error("%s is not valid! Must be one of %s", repr(backend), backend.keys())
+            continue
+        if backend not in enabled_backends:
+            enabled_backends.add(backend)
+
+    len_enabled_backends = len(enabled_backends)
+
     try:
-        return backend(query, sxng_locale)
+        results = []
+
+        for backend in enabled_backends:
+            backend_results = backend(query, sxng_locale)
+            if (len_enabled_backends > 1) and (len(backend_results) > 3):
+                # if more than 3 autocompleters: only get the first 3 results from each
+
+                results.extend(backend_results[:3:])
```

Comment:
Shouldn't this be
```suggestion
                results.extend(backend_results[:3])
```
?

Reply:
Yes, the result is the same but thats better.